gradle.properties

```diff
@@ -1,14 +1,26 @@
-org.gradle.jvmargs=-Xmx4g
+org.gradle.caching=true
+org.gradle.configureondemand=true
+org.gradle.jvmargs=-Xmx4g -Dfile.encoding=UTF-8
+org.gradle.parallel=true
 
 # AndroidX
 android.useAndroidX=true
 
+# Enable install debug apk manually
+android.injected.testOnly=false
```

Comment:
What does this do?

Reply:
Could let debuggale apk installed manually

Reply:
How does that increase build speed?

Reply:
No increasement, just convient for test